modules/react/common/lib/theming/useThemedRing.ts

```diff
@@ -54,10 +55,44 @@ export function getPaletteColors(
   }
 }
 
+/**
+ * This is a way to automatically add themed colors to your inputs. This is helpful when showing alerts
+ * to users. It supports `error`, `alert`, and `success` states. It will try and use the corresponding
+ * `main` colors from your `CanvasThemePalette` unless they do not meet accessibility contrast, in
+ * which case the outer ring will use the `darkest` color. This hook will also show a `focusOutline`
+ * ring when the input is focused. Note: You should not rely on these colors alone to differentiate
+ * alerts, but use them in combination with icons or hint text.
+ * @example
```

Comment:
Drop the `@example`. The doc system ignores anything after `@example`

```suggestion
 *
```